mlir/docs/Dialects/emitc.md

```diff
@@ -10,6 +10,7 @@ The following convention is followed:
     `emitc.call_opaque` operation, C++11 is required.
 *   If floating-point type template arguments are passed to an `emitc.call_opaque`
     operation, C++20 is required.
+*   If `ssize_t` is used, then the code requires the POSIX header `sys/types.h`,
```

Comment:
```suggestion
*   If `ssize_t` is used, then the code requires the POSIX header `sys/types.h`.
```

Reply:
Thanks for adding this to the markdown document.